pixart.js

```diff
@@ -1 +1,22 @@
+var colorInput = document.querySelector("#color-field");
+var brush = document.querySelector(".brush");
+var button = document.querySelector("#set-color");
 
+button.addEventListener("click", function(event){
+  event.preventDefault();
+  brush.style.backgroundColor = colorInput.value //why backgroundColor when the css file sets color by background?
```

Comment:
background is shorthand for a bunch of CSS properties that can also be declared individually. This is kind of like how we can say `margin: 0 10px 15px 5px;` and it means the same as:
```css
margin-top: 0;
margin-right: 10px;
margin-bottom: 15px;
margin-left: 5px;
```
There are a bunch of other CSS properties that have other shorthands like this where you can pass multiple arguments. For more info on background check out this [stack overflow article](http://stackoverflow.com/questions/10205464/what-is-the-difference-between-background-and-background-color#answer-10205500).